iFixit/react-commerce
# GA4: Install gtag tracking code

Our old GA account is sunsetting on June 30th. This old account is GA's Universal Analytics (UA). The new version is GA4.

Google seems to have made the transition easy enough:

You can run both the UA and GA4 code side by side
The existing events from the UA version are sent automagically

This will enable GA analytics (old and new)

QA
---
* Ensure you receive console logs for all the usual GA events
	* This tests the old UA is still working
* Ensure [the new debugviewer](https://analytics.google.com/analytics/web/?authuser=1#/a30506p383872824/admin/debugview/overview) shows the events sent to the GA4 instance
	* You may need to ping me for access to view this URL ^
* Also, ensure Matomo is receiving events as well

Connects: [#48129](https://github.com/iFixit/ifixit/issues/48129)
